---
source: tests/suite/check.rs
assertion_line: 35
expression: Snapshot(context)
---
Error: Semantic error
   ╭─[tests/check/bad-function-declarations1.xi:2:1]
   │
 2 │ foo() {}
   · ───┬───  
   ·    ╰───── Name already bound in environment
───╯

